assigner/commands/commit.py

```diff
@@ -113,6 +124,8 @@ def setup_parser(parser):
                         help="Include all changed files (i.e., git add -u or git commit -a)")
     parser.add_argument("-e", "--allow-empty", action="store_true", dest="allow_empty",
                         help="Commit even if there are no changes to commit")
+    parser.add_argument("-S", "--gpg-sign", action="store_true", dest="gpg_sign",
+                        help="GPG-sign the commits using the committer identity")
```

Comment:
It looks like `git commit` optionally allows the user to specify the gpg key id when using `-S` and friends; if you want to add that feature, you're more than welcome to, but it's not required.

Reply:
I had made a note of this in the PR description - I'm pretty sure it would be useless in this context because the commit verification works by making sure that the key ID/email matches the committer email (the default for -S).  The only use I can think of is if Alice had Bob's GPG privkey (both profs/graders/TAs) and wanted to make a change to a repo using Alice's email but Bob's signature (which would still pass verification I think).

Reply:
Ahh, yeah, that makes sense.